Setup/docker-compose.yml

```diff
@@ -71,28 +73,29 @@ services:
 #          - 5601:5601
 #          - 9200:9200
 #          - 5014:5014
-      networks:
-          - shield
-      environment:
-        - "REINDEX_RUNINNG_INTERVAL=30s"
-        - "CLEAN_RUNINNG_INTERVAL=12h"
-        - "LOGSPOUT=ignore"
-      deploy:
-        replicas: 1
-        restart_policy:
-            condition: any
-        placement:
-            constraints:
-              - node.labels.management==yes
-        update_config:
-          failure_action: rollback
-        resources:
-          limits:
-            memory: 6GB
-            cpus: "4"
-      volumes:
```

Comment:
This is production version and here we don't want reduce amount of memory and CPU.
We want increase instead. Even make separated node of ELK...